homework/lesson_number_2/lesson_number_2/Program.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace lesson_number_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] numbers = {5, 3, 1, 4, 2, 12, 15, 9, 8};
+
+            int[] a = SortArray(numbers);
+            int[] b = AddElementEndOfArray(numbers, 6);
+            int[] c = AddElementStartOfArray(numbers, 6);
+            int[] d = AddElementAnyPositionInArray(numbers, 6, 5);
+            int[] e = RemoveElementStartOfArray(numbers);
+            int[] f = RemoveElementEndOfArray(numbers);
+            int[] g = RemoveElementAnyPositionInArray(numbers, 2);
+          
+            //change the parameter below to show the outputs of the various array methods
+            PrintUpdatedArray(a);
+            
+            //send in correct username/password
+            string loginMsg = Login("Username", "Passw0rd");
+            Console.WriteLine(loginMsg);
+        }
+
+        private static void PrintUpdatedArray(int[] g)
+        {
+            for (int i = 0; i < g.Length; i++)
+            {
+                Console.WriteLine(g[i]);
+            }
+        }
+
+        private static string Login(string savedUsername, string savedPassword)
+        {
+            string loginMsg = "";
+            
+            Console.WriteLine("Please enter username: ");
+            string username = Console.ReadLine();
+
+            Console.WriteLine("Please enter password: ");
+            string password = Console.ReadLine();
+
+            if (username.Equals(savedUsername, StringComparison.OrdinalIgnoreCase) &&
+                password.Equals(savedPassword, StringComparison.Ordinal))
+            {
+                loginMsg = "Logged In!";
+                return loginMsg;
+            }
+
+            loginMsg = "Username or password was incorrect!";
+            return loginMsg;
+        }
+
+        static int[] SortArray(int[] numbers)
```

Comment:
I'd recommend looking up some sorting methods if you have time.  They can be interesting to learn about.  This method would not work if the array had duplicates.  You would end up adding some `999`'s into your array.